### DISCUSSION TITLE
How to stop it from Autoadding the workshop Folder as a mod directory?

### DISCUSSION THREAD

**Discussion by kingsly1** [X2CommunityCore/xcom2-launcher#267] [Help] [answered]
I wanna have two separate mod lists one for WotC and one for the base game but since it keeps re-adding the workshop mod folder I cant. any help would be greatly appreciatted

**Comment by RevZero** [comment, marked as answer]
To determine if the default folder was already added, AML checks if a path to a folder named `\268500` exists. So you could just add a dummy folder like `c:\temp\268500` and AML should no longer add the real default path, because it thinks that its already there.
